catalogue_pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/singlepagemiro/SierraMiroMergeRule.scala

```diff
@@ -0,0 +1,101 @@
+package uk.ac.wellcome.platform.merger.rules.singlepagemiro
+
+import grizzled.slf4j.Logging
+import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.platform.merger.logging.MergerLogging
+
+/** If we have a Miro work and a Sierra work with a single item,
+  * the Sierra work replaces the Miro work (because this is metadata
+  * that we can update).
+  *
+  * Specifically:
+  *
+  *   - We copy across all the identifiers from the Miro work (except
+  *     those which contain Sierra identifiers)
+  *   - We copy across the Miro location to the Sierra work *if* the
+  *     Sierra work doesn't already have a digital location.
+  *     In this case, the Sierra digital location supersedes the one
+  *     from Miro.
+  *
+  */
+object SierraMiroMergeRule extends SierraMiroMerger with SierraMiroPartitioner {
+  def mergeAndRedirectWork(works: Seq[BaseWork]): Seq[BaseWork] = {
```

Comment:
How do this method and the one below it with the same name interact? We have tended to avoid overloading elsewhere.

Reply:
Yeah that makes sense. I don't think there's a good reason